src/Sulu/Bundle/AdminBundle/Metadata/SchemaMetadata/SchemaMetadata.php

```diff
@@ -47,9 +47,18 @@ public function merge(self $schema): self
 
     public function toJsonSchema(): array
     {
+        $propertiesSchema = $this->propertiesMetadata->toJsonSchema();
+
+        if (!empty($propertiesSchema)) {
+            $propertiesSchema = \array_merge(
+                ['type' => 'object'],
+                $propertiesSchema
+            );
+        }
```

Comment:
This is necessary to remove the warning from ajv (because of strict mode, `properties` or `required` keyword must not appear without `"type": "object"`)

Reply:
would be nice to add the comment to the code